cmd/ore/aws/upload.go

```diff
@@ -122,6 +123,26 @@ func defaultBucketURL(urlPrefix, imageName, board, file, region string) (*url.UR
 	return s3URL, nil
 }
 
+type qemuInfo struct {
+	VirtualSize uint64 `json:"virtual-size"`
+}
+
+// We used to hardcode 8GB for Container Linux, but let's support
```

Comment:
Nit: history is what `git` is for.  :grin:

Reply:
Yeah, though I find it nice to avoid people needing to take that trip; when functions get moved around you need to traverse multiple layers of history, etc.